bluepyemodel/emodel_pipeline/emodel_settings.py

```diff
@@ -257,6 +266,35 @@ def __init__(
                 and EPSP protocols be run and plotted.
                 Should be True only for pyramidal cells,
                 since it depends on the presence of apical dendrite.
+            plot_IV_curves (bool): during the plotting, should peak voltage and voltage_deflection
+                IV curves be plotted for threshold-based sub-threshold IV protocols.
+                Needs pickle_cells_extraction to be set to True.
+            plot_FI_curve_comparison (bool): during the plotting, should FI curve be plotted
+                for experimental and simulated data.
+                Needs pickle_cells_extraction to be set to True.
+            plot_traces_comparison (bool): True to plot a new figure with simulated traces
+                on top of experimental traces.
+                Needs pickle_cells_extraction to be set to True.
+            run_plot_custom_sinspec (bool): True to run a SineSpec protocol, and plot
+            its voltage and current trace, along with its impedance.
+            IV_curve_prot_name (str): which protocol to use to plot_IV_curves.
+            FI_curve_prot_name (str): which protocol to use during plotting of FI curve comparison.
+                The protocol should be supra-threshold
+                and have the efeature mean_frequency associated to it.
+            plot_phase_plot (bool): during the plotting, should phase plot be plotted.
+            phase_plot_settings (dict): settings for the phase plot. Should contain the following:
+                "prot_names" (list of str): the names of the protocols to select for phase plot
+                "amplitude" (float): amplitude of the protocol to select.
+                    Only exactly this amplitude will be selected for model.
+                    An amplitude window is used for experimental trace selection
+                "amp_window" (float): amplitude window around amplitude for experimental
+                    recording selection.Is not used for model trace selection
```

Comment:
something seems missing here? between the two sentences

Reply:
Indeed, a space is missing. The second sentence does not have a subject because it is implied that amp_window is the subject.